modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -81,4 +97,233 @@ enter the specific IP address you want and related information. For example:
+
+a|`coreos.inst.save_partlabel`
+
+a|Optional: Comma-separated labels of partitions to preserve during the install. Glob-style wildcards are permitted. The specified partitions do not need to exist.
+
+a|`coreos.inst.save_partindex`
+
+a|Optional: Comma-separated indexes of partitions to preserve during the install. Ranges `m`-`n` are permitted, and either `m` or `n` can be omitted. The specified partitions do not need to exist.
+
+a|`coreos.inst.insecure`
+
+a|Optional: Permits the OS image to be unsigned, and installs without doing a gpgcheck. Must be used with `coreos.inst.image_url`.
+
+a|`coreos.inst.skip_reboot`
+
+a|Optional: The system will not reboot after installing. Once the install finishes, you will receive a prompt that allows you to inspect what is happening during installation. This argument should not be used in production environments and is intended for debugging purposes only.
+
+a|`ignition.config.url`
+
+a|Identifies the location of an Ignition config that you build to pass selected values to be consumed by the live installer. This argument can be used for adding disk partitioning or other features you want to have in place before the actual installation runs.
+
+|===
+
+[discrete]
+== `coreos-installer` options for ISO install
+
+The following table shows the options and subcommands you can pass to the `coreos-installer` command during an ISO live install.
+
+.`coreos-installer` options for ISO
+
+[source,adoc]
+|===
+|Option |Description
+
+a| -u, --image-url <url>
+
+a|Specify the image URL manually.
+
+a| -f, --image-file <path>
+
+a|Specify a local image file manually.
+
+a|-i, --ignition-file <path>
+
+a|Embed an Ignition config from a file.
+
+a|I, --ignition-url <URL>
+
+a|Embed an Ignition config from a URL.
+
+a|--ignition-hash <digest>
+
+a|Digest `type-value` of the Ignition config.
+
+a|-p, --platform <name>
+
+a|Override the Ignition platform ID.
+
+a|--append-karg <arg>...
+
+a|Append the default kernel argument.
+
+a|--delete-karg <arg>...
+
+a|Delete the default kernel argument.
+
+a|-n, --copy-network
+
+a|Copy the network configuration from the install environment.
+
+a|--network-dir <path>
+
+a|For use with `-n`. Default is `/etc/NetworkManager/system-connections/`.
+
+a|--save-partlabel <lx>..
+
+a|Save partitions with this label glob.
+
+a|--save-partindex <id>...
+
+a|Save partitions with this number or range.
+
+a|--offline
+
+a|Force offline installation.
+
+a|--insecure
+
+a|Skip signature verification.
+
+a|--insecure-ignition
+
+a|Allow Ignition URL without HTTPS or hash.
+
+a|--architecture <name>
+
+a|Target CPU architecture. Default is `x86_64`.
+
+a|--preserve-on-error
+
+a|Do not clear partition table on error.
+
+a|-h, --help
+
+a|Print help information.
+|===
+
+|===
+|Argument |Description
+
+a|<device>
+
+a|The destination device.
+
+|===
+
+[discrete]
+=== `coreos-installer` ISO Ignition examples
```

Comment:
These were included towards the end of the G doc. I'm not sure if they should be included this way. And if so, would it make more sense to break out as a procedure module?